### PR TITLE
Bulkloadcases

### PR DESCRIPTION
**Pull request by SarahRadford**
Please can you try out the code following the instructions in the readme.

### CODE REVIEW

File: case-loader/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+# Load Data
+The loadcases.py program will load the CSVs created into the relevant Postgres tables. The data in the tables is cleared before the new data is loaded.
+
+NOTE: No IACs will be created as the case delivery queue is bypassed.
+
+## Setting up the python environment
+```
+brew install pyenv
+pyenv install
```
Comment (cookel2) [inline case-loader/README.md]: I'm not sure that this command is needed:
pyenv install

When I tried to use it it gave me the following usage message:

Usage: pyenv install [-f] [-kvp] <version>
       pyenv install [-f] [-kvp] <definition-file>
       pyenv install -l|--list
       pyenv install --version

  -l/--list          List all available versions
  -f/--force         Install even if the version appears to be installed already
  -s/--skip-existing Skip if the version appears to be installed already

  python-build options:

  -k/--keep          Keep source tree in $PYENV_BUILD_ROOT after installation
                     (defaults to $PYENV_ROOT/sources)
  -p/--patch         Apply a patch from stdin before building
  -v/--verbose       Verbose mode: print compilation status to stdout
  --version          Show version of python-build
  -g/--debug         Build a debug version

For detailed information on installing Python versions with
python-build, including a list of environment variables for adjusting
compilation, see: https://github.com/pyenv/pyenv#readme

File: case-loader/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+pip install --upgrade pip setuptools pipenv
```
Comment (cookel2) [inline case-loader/README.md]: It didn't like this command either:

pip install --upgrade pip setuptools pipenv

It gave me the following error message:

-bash: pip: command not found

**Comment by cookel2** [comment]
Sarah, I have pulled down the code into my local environment and followed the steps in the readme file.  The createcsvs.py ran successfully but the loadcases.py gave me the following error:

clearing postgres tables...
Traceback (most recent call last):
  File "loadcases.py", line 15, in <module>
    cur.execute("delete from action.actionrule")
psycopg2.IntegrityError: update or delete on table "actionrule" violates foreign key constraint "actionrulefk_fkey" on table "action"
DETAIL:  Key (actionrulepk)=(6) is still referenced from table "action".

I also thought it would be good to say that the steps in the last section need to be followed from inside the folder named case-loader (I tried following them from the outer folder first by mistake). 

I hope this is useful feedback! Thanks, Ella

**Comment by cookel2** [comment]
Sarah, I have now updated my postgres database, as it was not as up to date as yours, and have followed the updated readme instructions too; it all works beautifully now. 
Ella

**Comment by SarahRadford** [comment]
Thanks Ella

**Pull request merged**